<?xml version="1.0" encoding="utf-8" ?>
<AnalysisProgramOfRegionAnalysisSessionString xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <SaveName>BuildFailureReason/Scala/scala@scala</SaveName>
  <LearningData>
    <InputPaths>
      <string>Scala/scala@scala/failed/566675131.log</string>
      <string>Scala/scala@scala/failed/566726054.log</string>
      <string>Scala/scala@scala/failed/566805126.log</string>
      <string>Scala/scala@scala/failed/566852148.log</string>
      <string>Scala/scala@scala/failed/566866822.log</string>
      <string>Scala/scala@scala/failed/567060744.log</string>
      <string>Scala/scala@scala/failed/567103817.log</string>
      <string>Scala/scala@scala/failed/567119813.log</string>
      <string>Scala/scala@scala/failed/567174876.log</string>
      <string>Scala/scala@scala/failed/567184269.log</string>
    </InputPaths>
    <Examples>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/566675131.log</InputPath>
        <Keywords>!!, failed, error</Keywords>
        <Category>0</Category>
        <Output>!![0m[0m  753 - neg/t742.scala                            [compilation failed]
[1m[36m##### Log file '/home/travis/build/scala/scala/test/files/neg/t742-neg.log' from failed test #####
[0m[0m
error: head of empty list</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/566726054.log</InputPath>
        <Keywords>!!, failed, error</Keywords>
        <Category>0</Category>
        <Output>!![0m[0m  754 - neg/t742.scala                            [compilation failed]
[1m[36m##### Log file '/home/travis/build/scala/scala/test/files/neg/t742-neg.log' from failed test #####
[0m[0m
error: head of empty list</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/566805126.log</InputPath>
        <Keywords>error, failed, Exception</Keywords>
        <Category>1</Category>
        <Output>error[0m] [0m1 of 13 test tasks failed:[0m
[0m[[31merror[0m] [0m- reflect/mimaReportBinaryIssues[0m
[0m[[31merror[0m] [0m  - reflect/mimaReportBinaryIssues failed: java.lang.RuntimeException: scala-reflect: Binary compatibility check failed![0m
java.lang.RuntimeException
	at $b0329634e469e03f667a$$anonfun$root$27.apply(build.sbt:1031)
	at $b0329634e469e03f667a$$anonfun$root$27.apply(build.sbt:973)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[0m[[31merror[0m] [0m(root/*:[31mtestAll[0m) java.lang.RuntimeException[0m
[0m[[31merror[0m] [0mTotal time: 4043 s, completed Aug 2, 2019 7:27:52 AM[0m</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/566852148.log</InputPath>
        <Keywords>error, failed, </Keywords>
        <Category>1</Category>
        <Output>error[0m] [0m[0m [0m
[0m[[0m[31merror[0m] [0m[0m1 of 17 test tasks failed:[0m
[0m[[0m[31merror[0m] [0m[0m- library/mimaReportBinaryIssues[0m
[0m[[0m[31merror[0m] [0m[0m  - library/mimaReportBinaryIssues failed: java.lang.RuntimeException: scala-library: Binary compatibility check failed![0m
[0m[[0m[31merror[0m] [0m[0mSaved log of library/mimaReportBinaryIssues:[0m
[0m[[0m[31merror[0m] [0m[0m * method processUnicode(java.lang.String)java.lang.String in object scala.StringContext does not have a correspondent in other version[0m
[0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[DirectMissingMethodProblem]("scala.StringContext.processUnicode")[0m
[0m[[0m[31merror[0m] [0m[0mtestAll failed due to previous errors[0m
[0m[[0m[31merror[0m] [0m[0m([31mtestAll[0m) testAll failed due to previous errors[0m</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/566866822.log</InputPath>
        <Keywords>error, failed, </Keywords>
        <Category>1</Category>
        <Output>error[0m] [0m[0m [0m
[0m[[0m[31merror[0m] [0m[0m1 of 17 test tasks failed:[0m
[0m[[0m[31merror[0m] [0m[0m- library/mimaReportBinaryIssues[0m
[0m[[0m[31merror[0m] [0m[0m  - library/mimaReportBinaryIssues failed: java.lang.RuntimeException: scala-library: Binary compatibility check failed![0m
[0m[[0m[31merror[0m] [0m[0mSaved log of library/mimaReportBinaryIssues:[0m
[0m[[0m[31merror[0m] [0m[0m * method processUnicode(java.lang.String)java.lang.String in object scala.StringContext does not have a correspondent in other version[0m
[0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[DirectMissingMethodProblem]("scala.StringContext.processUnicode")[0m
[0m[[0m[31merror[0m] [0m[0mtestAll failed due to previous errors[0m
[0m[[0m[31merror[0m] [0m[0m([31mtestAll[0m) testAll failed due to previous errors[0m</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/567060744.log</InputPath>
        <Keywords>error, failed, </Keywords>
        <Category>1</Category>
        <Output>error[0m] [0m[0m [0m
[0m[[0m[31merror[0m] [0m[0m2 of 17 test tasks failed:[0m
[0m[[0m[31merror[0m] [0m[0m- partest run[0m
[0m[[0m[31merror[0m] [0m[0m  - test/it:testOnly failed: sbt.TestsFailedException: Tests unsuccessful[0m
[0m[[0m[31merror[0m] [0m[0m- library/mimaReportBinaryIssues[0m
[0m[[0m[31merror[0m] [0m[0m  - library/mimaReportBinaryIssues failed: java.lang.RuntimeException: scala-library: Binary compatibility check failed![0m
[0m[[0m[31merror[0m] [0m[0mSaved log of library/mimaReportBinaryIssues:[0m
[0m[[0m[31merror[0m] [0m[0m * abstract method stepper(scala.collection.StepperShape)scala.collection.Stepper in class scala.collection.immutable.ArraySeq does not have a correspondent in current version[0m
[0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[DirectAbstractMethodProblem]("scala.collection.immutable.ArraySeq.stepper")[0m
[0m[[0m[31merror[0m] [0m[0m * in current version there is abstract method stepper(scala.collection.StepperShape)scala.collection.Stepper in class scala.collection.immutable.ArraySeq, which does not have a correspondent[0m
[0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[ReversedAbstractMethodProblem]("scala.collection.immutable.ArraySeq.stepper")[0m
[0m[[0m[31merror[0m] [0m[0m * abstract method stepper(scala.collection.StepperShape)scala.collection.Stepper in class scala.collection.mutable.ArraySeq does not have a correspondent in current version[0m
[0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[DirectAbstractMethodProblem]("scala.collection.mutable.ArraySeq.stepper")[0m
[0m[[0m[31merror[0m] [0m[0m * in current version there is abstract method stepper(scala.collection.StepperShape)scala.collection.Stepper in class scala.collection.mutable.ArraySeq, which does not have a correspondent[0m
[0m[[0m[31merror[0m] [0m[0m   filter with: ProblemFilters.exclude[ReversedAbstractMethodProblem]("scala.collection.mutable.ArraySeq.stepper")[0m
[0m[[0m[31merror[0m] [0m[0mtestAll failed due to previous errors[0m
[0m[[0m[31merror[0m] [0m[0m([31mtestAll[0m) testAll failed due to previous errors[0m</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/567103817.log</InputPath>
        <Keywords>error, Exception, </Keywords>
        <Category>1</Category>
        <Output>sbt.ResolveException: unresolved dependency: org.scala-lang#scala-compiler;2.12.9-bin-5d0a905-SNAPSHOT: not found
	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:320)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:191)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:168)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156)
	at sbt.IvySbt$$anonfun$withIvy$1.apply(Ivy.scala:133)
	at sbt.IvySbt.sbt$IvySbt$$action$1(Ivy.scala:57)
	at sbt.IvySbt$$anon$4.call(Ivy.scala:65)
	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:93)
	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:78)
	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:97)
	at xsbt.boot.Using$.withResource(Using.scala:10)
	at xsbt.boot.Using$.apply(Using.scala:9)
	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:58)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:48)
	at xsbt.boot.Locks$.apply0(Locks.scala:31)
	at xsbt.boot.Locks$.apply(Locks.scala:28)
	at sbt.IvySbt.withDefaultLogger(Ivy.scala:65)
	at sbt.IvySbt.withIvy(Ivy.scala:128)
	at sbt.IvySbt.withIvy(Ivy.scala:125)
	at sbt.IvySbt$Module.withModule(Ivy.scala:156)
	at sbt.IvyActions$.updateEither(IvyActions.scala:168)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1555)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1551)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$122.apply(Defaults.scala:1586)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$122.apply(Defaults.scala:1584)
	at sbt.Tracked$$anonfun$lastOutput$1.apply(Tracked.scala:37)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1589)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1583)
	at sbt.Tracked$$anonfun$inputChanged$1.apply(Tracked.scala:60)
	at sbt.Classpaths$.cachedUpdate(Defaults.scala:1606)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1533)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1485)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
sbt.ResolveException: unresolved dependency: org.scala-lang#scala-compiler;2.12.9-bin-5d0a905-SNAPSHOT: not found
	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:320)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:191)
	at sbt.IvyActions$$anonfun$updateEither$1.apply(IvyActions.scala:168)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:156)
	at sbt.IvySbt$$anonfun$withIvy$1.apply(Ivy.scala:133)
	at sbt.IvySbt.sbt$IvySbt$$action$1(Ivy.scala:57)
	at sbt.IvySbt$$anon$4.call(Ivy.scala:65)
	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:93)
	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:78)
	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:97)
	at xsbt.boot.Using$.withResource(Using.scala:10)
	at xsbt.boot.Using$.apply(Using.scala:9)
	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:58)
	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:48)
	at xsbt.boot.Locks$.apply0(Locks.scala:31)
	at xsbt.boot.Locks$.apply(Locks.scala:28)
	at sbt.IvySbt.withDefaultLogger(Ivy.scala:65)
	at sbt.IvySbt.withIvy(Ivy.scala:128)
	at sbt.IvySbt.withIvy(Ivy.scala:125)
	at sbt.IvySbt$Module.withModule(Ivy.scala:156)
	at sbt.IvyActions$.updateEither(IvyActions.scala:168)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1555)
	at sbt.Classpaths$$anonfun$sbt$Classpaths$$work$1$1.apply(Defaults.scala:1551)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$122.apply(Defaults.scala:1586)
	at sbt.Classpaths$$anonfun$doWork$1$1$$anonfun$122.apply(Defaults.scala:1584)
	at sbt.Tracked$$anonfun$lastOutput$1.apply(Tracked.scala:37)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1589)
	at sbt.Classpaths$$anonfun$doWork$1$1.apply(Defaults.scala:1583)
	at sbt.Tracked$$anonfun$inputChanged$1.apply(Tracked.scala:60)
	at sbt.Classpaths$.cachedUpdate(Defaults.scala:1606)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1533)
	at sbt.Classpaths$$anonfun$updateTask$1.apply(Defaults.scala:1485)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[0m[[31merror[0m] [0m(root/*:[31mupdate[0m) sbt.ResolveException: unresolved dependency: org.scala-lang#scala-compiler;2.12.9-bin-5d0a905-SNAPSHOT: not found[0m
[0m[[31merror[0m] [0m(bootstrap/*:[31mupdate[0m) sbt.ResolveException: unresolved dependency: org.scala-lang#scala-compiler;2.12.9-bin-5d0a905-SNAPSHOT: not found[0m
[0m[[31merror[0m] [0mTotal time: 7 s, completed Aug 2, 2019 8:05:50 PM[0m</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/567119813.log</InputPath>
        <Keywords>error, , </Keywords>
        <Category>1</Category>
        <Output>error[0m] [0m/home/travis/build/scala/scala/src/reflect/scala/reflect/internal/ReificationSupport.scala:614: ')' expected but 'else' found.[0m
[0m[[31merror[0m] [0m        if (sumSize(unlifted.flatten, 0) == sumSize(lst.flatten, 0) Some(unlifted) else None[0m
[0m[[31merror[0m] [0m                                                                                   ^[0m
[0m[[31merror[0m] [0mone error found[0m
[0m[[31merror[0m] [0m(reflect/compile:[31mcompileIncremental[0m) Compilation failed[0m
[0m[[31merror[0m] [0mTotal time: 120 s, completed Aug 2, 2019 8:51:01 PM[0m</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/567174876.log</InputPath>
        <Keywords>error, failed, </Keywords>
        <Category>1</Category>
        <Output>error[0m] [0m1 of 13 test tasks failed:[0m
[0m[[31merror[0m] [0m- partest instrumented presentation[0m
[0m[[31merror[0m] [0m  - test/it:testOnly failed: sbt.TestsFailedException: Tests unsuccessful[0m
java.lang.RuntimeException
	at $e1fdba7da5812f40a13c$$anonfun$root$27.apply(build.sbt:1031)
	at $e1fdba7da5812f40a13c$$anonfun$root$27.apply(build.sbt:973)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[0m[[31merror[0m] [0m(root/*:[31mtestAll[0m) java.lang.RuntimeException[0m
[0m[[31merror[0m] [0mTotal time: 3725 s, completed Aug 3, 2019 1:30:08 AM[0m</Output>
      </ExampleDataOfString>
      <ExampleDataOfString>
        <InputPath>Scala/scala@scala/failed/567184269.log</InputPath>
        <Keywords>error, failed, </Keywords>
        <Category>1</Category>
        <Output>error[0m] [0m1 of 13 test tasks failed:[0m
[0m[[31merror[0m] [0m- reflect/mimaReportBinaryIssues[0m
[0m[[31merror[0m] [0m  - reflect/mimaReportBinaryIssues failed: java.lang.RuntimeException: scala-reflect: Binary compatibility check failed![0m
java.lang.RuntimeException
	at $5752eebc85ce39959579$$anonfun$root$27.apply(build.sbt:1031)
	at $5752eebc85ce39959579$$anonfun$root$27.apply(build.sbt:973)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[0m[[31merror[0m] [0m(root/*:[31mtestAll[0m) java.lang.RuntimeException[0m
[0m[[31merror[0m] [0mTotal time: 3910 s, completed Aug 3, 2019 2:28:02 AM[0m</Output>
      </ExampleDataOfString>
    </Examples>
  </LearningData>
  <LogKind>
    <Language>java</Language>
    <BuildTool>android</BuildTool>
  </LogKind>
  <Target>BuildFailureReason</Target>
</AnalysisProgramOfRegionAnalysisSessionString>